src/scandeval/benchmark_dataset.py

```diff
@@ -46,6 +47,15 @@
 # Set up logger
 logger = logging.getLogger(__name__)
 
+def _local_model_config(model_id: str, benchmark_config: BenchmarkConfig):
+    framework = benchmark_config.model_framework
+    if framework is None:
+        if benchmark_config.raise_errors:
+            raise RuntimeError(f"No framework specified for local model `{model_id}`!")
+        framework = "pytorch"
+    model_config = ModelConfig(model_id=model_id, revision="local", framework=framework, task="unknown", languages=[])
```

Comment:
Just using the same defaults as for fresh models here:

```suggestion
    model_config = ModelConfig(model_id=model_id, revision="main", framework=framework, task="fill-mask", languages=[])
```